### PR TITLE
Jal/readme.md route/ingress upd

### PR DESCRIPTION
**Pull request by jleslie85**
updates for on-cluster Insights deployed on managed K8s (port-forward command).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -302,11 +302,16 @@ done
...
-echo "Open Networking/Route panel in your Kubernetes dashboard to obtain your NuoDB Insight's dashboard URL."
-echo ""
-echo "https://$(kubectl get route grafana-route --output=jsonpath={.spec.host})//d/000000002/system-overview?orgId=1&refresh=10s"
-
+# if Red Hat OpenShift,
+# open your Kubernetes dashboard Networking/Route panel to obtain your NuoDB Insight's dashboard URL.
+# Also, displayed using the following command,
+   echo "on-cluster Insights URL: https://$(kubectl get route grafana-route --output=jsonpath={.spec.host})//d/000000002/system-overview?orgId=1&refresh=10s"
+
+# if managed Kubernetes / open source Kubernetes,
```
Comment (thebithead) [inline README.md]: You only need to use the port-forward if the Route or Ingress didn't work.

File: README.md [hunk 2]
```diff
@@ -381,26 +386,37 @@ Optionally deploy the NuoDB Insights visual monitoring tool **(recommended)**. I
       *insightsEnabled* to "Opt In" and enable NuoDB Insights. Any other value than "true"
       results in Opting out. Insights can also be enabled at a later time if you choose.
 
-After deploying your NuoDB database, if you optionally chose to install NuoDB Insights, you can find your NuoDB Insights SubcriberID by locating the "nuodb-insights" pod, go to the Logs tab, and find the line that indicates your Subscriber ID. 
+After deploying your NuoDB database, if you optionally chose to install NuoDB Insights by setting "insightsEnabled: true" in your nuodb-cr.yaml file and applying the &ensp;`nuodbinsightsserver_crd.yaml` and &ensp;`nuodbinsightsserver_cr.yaml`, then the default is to install "on-cluster" NuoDB Insights. With this option, all performance data is privately stored and managed locally on your cluster by starting local elasticsearch, logstash, kibana, and grafana components that are utilized by the Insights monitoring solution.
```
Comment (thebithead) [inline README.md]: That's not correct.  The "insightsEnabled" that is currently in the NuoDB CR is only used for our "hosted" NuoDB Insights Solution, not the "on-cluster" NuodbInsightsServer.  There will be fixed in Jira DB-28696

File: README.md [hunk 3]
```diff
@@ -256,7 +284,7 @@ sed "s/placeholder/$OPERATOR_NAMESPACE/" nuodb-operator/deploy/olm-catalog/nuodb
 # To pull from the Google Marketplace, run
 #   sed "s/quay.io/marketplace.gcr.io/"          nuodb-csv.yaml > nuodb-csv-gcp.yaml
 # To pull from the AWS Marketplace, 
-# replace in the nuodb-csv.yaml file the two image references wtih 709373726912.dkr.ecr.us-east-1.amazonaws.com/d893f8e5-fe12-4e43-b792-8cb98ffc11c0/cg-1180980994/quay.io/nuodb/nuodb-operator:0.0.5-latest
+# replace in the nuodb-csv.yaml file the two image references with 709373726912.dkr.ecr.us-east-1.amazonaws.com/d893f8e5-fe12-4e43-b792-8cb98ffc11c0/cg-1180980994/quay.io/nuodb/nuodb-operator:0.0.5-latest
```
Comment (thebithead) [inline README.md]: We started at version v2.0.2.

File: README.md [hunk 4]
```diff
@@ -32,12 +32,12 @@ This page is organized in the following sections:
 
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Installation Prerequisites](#Installation-Prerequisites)
```
Comment (vegichan) [inline README.md]: this is a general comment.

When we discussed markdown formatting guidelines for NuoDB docs, we said that each sentence should be on its own line, to make reviews of text easier.

I have tried to follow that guidance in the Helm charts.

Maybe we want to do something like that in this repo as well.

File: README.md [hunk 5]
```diff
@@ -180,6 +180,34 @@ kubectl  create secret docker-registry pull-secret \
 kubectl create -n $OPERATOR_NAMESPACE -f nuodb-operator/deploy/thp-scc.yaml
 ```
 
+## Configure NuoDB Insights Visual Monitor
+
+Optionally deploy the NuoDB Insights visual monitoring tool **(recommended)**. NuoDB Insights is a powerful database monitoring tool that can greatly aid in visualizing database workload and resource consumption. For more information about the benefits of Insights, please refer to the [NuoDB Insights](https://www.nuodb.com/product/insights) Webpage.
+
+> Insights is also part of NuoDB Services and Support in order to service our customers better and more efficiently and is
```
Comment (vegichan) [inline README.md]: I feel like this should be "Insights is also part of NuoDB Services and Support. In..."

File: README.md [hunk 6]
```diff
@@ -180,6 +180,34 @@ kubectl  create secret docker-registry pull-secret \
...
+      addresses).
+      NuoDB uses Usage Information to monitor, analyze and improve the performance
+      and reliability of our Services, and to contribute to analytical models used by
+      NuoDB.  Usage Information is not shared with any third parties.  Insights also
+      includes a user dashboard that allows administrators to view the performance of
+      your NuoDB implementation.
+      If you agree to these terms, following the below instructions to enable NuoDB Insights.
+      Insights can also be enabled at a later time if you choose.
+
+Before deploying  NuoDB, to enable NuoDB Insights you can
```
Comment (vegichan) [inline README.md]: double space between deploying and NuoDB

File: README.md [hunk 7]
```diff
@@ -275,54 +303,20 @@ done
...
 
-The nuodb-operator/deploy directory includes sample Custom Resources to deploy NuoDB:
+The nuodb-operator/deploy directory includes sample Custom Resources to deploy the NuoDB database:
 
 &ensp; `cr-ephemeral.yaml` deploys NuoDB CE domain without a persistent storage volume by setting storageMode to "ephemeral".
 
 &ensp; `cr-persistent-insights-enabled.yaml` deploys NuoDB CE domain using persistent storage and has NuoDB Insights enabled.
 
-Optionally, you can add any of these below parameters values to your own `nuodb-cr.yaml` to customize your database. Each are described in the&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Optional Database Parameters](#Optional-Database-Parameters) section. A sample nuodb-ycsb-cr.yaml is also provided. The nuodb-insights-cr.yaml normally does not require modification.
+Optionally, you can add any of these below parameters values to your own `nuodb-cr.yaml` to customize your database. Each are described in the &nbsp;[Optional Database Parameters](#Optional-Database-Parameters) section. A sample nuodb-ycsb-cr.yaml is also provided. The nuodb-insights-cr.yaml normally does not require modification.
```
Comment (vegichan) [inline README.md]: "any of these below parameters values" reads funny

**Pull request merged**